clang/lib/CIR/CodeGen/CIRGenStmtOpenACC.cpp

```diff
@@ -32,46 +32,52 @@ constexpr bool isOneOfTypes =
 template <typename ToTest, typename T>
 constexpr bool isOneOfTypes<ToTest, T> = std::is_same_v<ToTest, T>;
 
+template <typename OpTy>
```

Comment:
We end up needing to template-ize this, since the visitor handler functions are not needing to `if-constexpr` instead of just the `apply` function.